src/alert/alert.scss

```diff
@@ -17,43 +18,54 @@
     background-color: t(iui-color-background-1);
   }
 
-  &-icon {
-    @include iui-icons-default;
-    flex-shrink: 0;
-    margin-left: $iui-m;
-  }
-
-  &-message {
-    margin: $iui-baseline $iui-m;
-  }
-
-  &-link {
-    @include iui-anchor-underline('on-initial');
-    border-radius: $iui-border-radius;
-    cursor: pointer;
-    margin-left: $iui-s;
-    user-select: none;
-    white-space: nowrap;
-  }
-
-  > .iui-button {
-    margin-left: auto;
-    margin-right: $iui-s;
-  }
-
   &.iui-sticky {
     border-radius: 0;
-    position: absolute;
+    position: sticky;
     top: 0;
     left: 0;
-    width: 100vw;
+    display: flex;
+    z-index: 1;
```

Comment:
![image](https://user-images.githubusercontent.com/78564221/166729185-91676a0a-4288-49a1-abb0-af8f9e11cf93.png)

Came across this issue when the sticky alert was over another alert. It seems `z-index: 1` solves it.

Reply:
Not sure we should control that.. Would assume its on user in what container sticky alert is put.. Or I misunderstood the issue

Reply:
I agree. Removed